esmvalcore/_data_finder.py

```diff
@@ -17,7 +17,7 @@
 
 def find_files(dirnames, filenames):
     """Find files matching filenames in dirnames."""
-    logger.debug("Looking for files matching %s in %s", filenames, dirnames)
+    logger.info("Looking for files matching %s in %s", filenames, dirnames)
```

Comment:
This should probably stay a debug message, because it will generate a huge amount of output